github-for-jira.sd.yml

```diff
@@ -240,7 +286,20 @@ environmentOverrides:
           instance: t2.small
           min: 1
           max: 5
-          metrics: *CpuMemScalingRules
+          metrics: *CpuMemAndQueuesScalingRules
+    resources:
```

Comment:
I had to add this resource because without it we are loosing DNS name which we used for dev environment.